<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>812</width>
    <height>639</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Départ</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="3">
     <widget class="QComboBox" name="leVilleDepart"/>
    </item>
    <item row="0" column="6">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Arrivée</string>
      </property>
     </widget>
    </item>
    <item row="0" column="7">
     <widget class="QComboBox" name="leVilleArrivee"/>
    </item>
    <item row="0" column="9">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Date départ</string>
      </property>
     </widget>
    </item>
    <item row="0" column="11" colspan="2">
     <widget class="QDateEdit" name="dateEdit">
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="3">
     <widget class="QPushButton" name="pbRechercheListVols">
      <property name="text">
       <string>Rechercher les vols</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="13">
     <widget class="QTableWidget" name="twListeDesVols">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="showGrid">
       <bool>false</bool>
      </property>
      <property name="gridStyle">
       <enum>Qt::NoPen</enum>
      </property>
      <property name="rowCount">
       <number>0</number>
      </property>
      <property name="columnCount">
       <number>9</number>
      </property>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QPushButton" name="pbValiderUnVol">
      <property name="text">
       <string>Valider</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="13">
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>ID client</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1" colspan="2">
     <widget class="QLineEdit" name="leIDPassager"/>
    </item>
    <item row="5" column="3">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Nom</string>
      </property>
     </widget>
    </item>
    <item row="5" column="4" colspan="2">
     <widget class="QLineEdit" name="leNomPassager"/>
    </item>
    <item row="5" column="6">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Prenom</string>
      </property>
     </widget>
    </item>
    <item row="5" column="7">
     <widget class="QLineEdit" name="lePrenomPassager"/>
    </item>
    <item row="5" column="8">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Tél</string>
      </property>
     </widget>
    </item>
    <item row="5" column="9">
     <widget class="QLineEdit" name="leTelephonePassager"/>
    </item>
    <item row="5" column="10">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Email</string>
      </property>
     </widget>
    </item>
    <item row="5" column="11" colspan="2">
     <widget class="QLineEdit" name="leEmailPassager"/>
    </item>
    <item row="6" column="0" colspan="13">
     <widget class="QTableWidget" name="twInformationPassager">
      <property name="showGrid">
       <bool>false</bool>
      </property>
      <property name="gridStyle">
       <enum>Qt::SolidLine</enum>
      </property>
      <property name="columnCount">
       <number>5</number>
      </property>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
     </widget>
    </item>
    <item row="7" column="0" colspan="3">
     <widget class="QPushButton" name="RechercherPassager">
      <property name="text">
       <string>Rechercher</string>
      </property>
     </widget>
    </item>
    <item row="7" column="7">
     <widget class="QPushButton" name="AjouterPassager">
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
    </item>
    <item row="7" column="11" colspan="2">
     <widget class="QPushButton" name="ModifierPassager">
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0" colspan="13">
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>ID vol</string>
      </property>
     </widget>
    </item>
    <item row="10" column="0">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>ID client</string>
      </property>
     </widget>
    </item>
    <item row="11" column="0">
     <widget class="QLabel" name="label_13">
      <property name="text">
       <string>ID Siège</string>
      </property>
     </widget>
    </item>
    <item row="12" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>N° billet: </string>
      </property>
     </widget>
    </item>
    <item row="12" column="1" colspan="6">
     <widget class="QLineEdit" name="leReserv_UUID">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="12" column="7" colspan="2">
     <widget class="QPushButton" name="pbCreer_Reservation">
      <property name="text">
       <string>Reserver</string>
      </property>
     </widget>
    </item>
    <item row="12" column="9">
     <widget class="QPushButton" name="pbRechercher_Reservation">
      <property name="text">
       <string>Rechercher</string>
      </property>
     </widget>
    </item>
    <item row="12" column="10" colspan="2">
     <widget class="QPushButton" name="pbModifier_Reservation">
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
    </item>
    <item row="12" column="12">
     <widget class="QPushButton" name="pbAnnuler_Reservation">
      <property name="text">
       <string>Annuler</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1" colspan="2">
     <widget class="QLineEdit" name="leReserv_IDVOL">
      <property name="enabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="10" column="1" colspan="2">
     <widget class="QLineEdit" name="leReserv_IDPassager">
      <property name="enabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="11" column="1" colspan="2">
     <widget class="QLineEdit" name="leIDSiegeReserv">
      <property name="enabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="9" column="3" rowspan="3" colspan="10">
     <widget class="QTableWidget" name="twInformation_Reservation">
      <property name="showGrid">
       <bool>true</bool>
      </property>
      <property name="columnCount">
       <number>12</number>
      </property>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>812</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>RightToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
